docs/protocol/core/v3.0.rst

```diff
@@ -103,11 +103,72 @@ Data types
 
 TODO define core data types
 
-Regular chunk grids
--------------------
-
-TODO define regular chunk grids, including how to form a key for each chunk in a grid
-
+Chunk grids
+-----------
+
+A chunk grid defines a set of chunks which contain the elements of an
+array. The chunks of a grid form a tessellation of the array space,
```

Comment:
Do you foresee a _simpler_ grid definition than doesn't include the relationship to an array?

Reply:
Ah OK. IIUC what @axtimwalde was saying was that there are some useful functionalities that could be provided without needing to know anything about the grid layout. For example, if you wanted to copy data from one store to another, or recode chunks using a different compressor. But I think the core protocol needs to define the full picture, in the sense that chunks always belong to an array, and that array will have some grid layout that defines what the chunks contain.